src/AddTodoForm.js

```diff
@@ -1,22 +1,33 @@
 import React from 'react';
 
-const AddTodoForm = (props) => {
+const AddTodoForm = ({onAddTodo}) => {
+
+    const [todoTitle, setTodoTitle] = React.useState('');
+
+    const handleTitleChange = (event) => {
+        const newTodoTitle = event.target.value;
+        setTodoTitle(newTodoTitle);
+    }
 
     const handleAddTodo = (event) => {
         event.preventDefault();
-        const todoTitle = event.target.title.value;
-        props.onAddTodo(todoTitle);
+        // const todoTitle = event.target.title.value;
```

Comment:
good job, these commented out lines would be alternate ways to achieve the same results, but the assignment wants to do the state method for this one

Reply:
Thank you, Darga. Deleted.